Push headers from R at the end to avoid extraneous overwrites
author: Alex Domingo (Vrije Universiteit Brussel)
--- src/R_init_rods.cpp.orig	2021-03-03 00:06:08.472875000 +0100
+++ src/R_init_rods.cpp	2021-03-03 00:11:51.625664000 +0100
@@ -1,10 +1,10 @@
+#include "irods_client_api_table.hpp"
+#include "irods_pack_table.hpp"
+
 #include <R.h>
 #include <Rinternals.h>
 #include <R_ext/Rdynload.h>
 
-#include "irods_client_api_table.hpp"
-#include "irods_pack_table.hpp"
-
 void
 R_init_rods(DllInfo *info)
 {
